docs-source/content/samples/simple/tanzu-kubernetes-service/_index.md

```diff
@@ -0,0 +1,763 @@
+   
+   b. Find and then pull the WebLogic 12.2.1.4 install image:
+   
+      ```bash
+      $ docker pull container-registry.oracle.com/middleware/weblogic:12.2.1.4
+      ```
+
+#### Install WebLogic Server Kubernetes Operator
+
+The Oracle WebLogic Server Kubernetes Operator is an adapter to integrate WebLogic Server and Kubernetes, allowing Kubernetes to serve as a container infrastructure hosting WLS instances. 
+The operator runs as a Kubernetes Pod and stands ready to perform actions related to running WLS on Kubernetes.
+
+Clone the Oracle WebLogic Server Kubernetes Operator repository to your machine. We will use several scripts in this repository to create a WebLogic domain. This sample was tested with v3.0.3
+Kubernetes Operators, use [Helm](https://helm.sh/) to manage Kubernetes applications. The operator’s Helm chart is located in the `kubernetes/charts/weblogic-operator` directory. Please install the operator by running the following commands.
+
+Clone repository
+
+```bash
+$ git clone https://github.com/oracle/weblogic-kubernetes-operator.git
+$ cd weblogic-kubernetes-operator
+$ git checkout v3.0.3
+```
+
+Grant the Helm service account the cluster-admin role.
+
+```bash
+$ cat <<EOF | kubectl apply -f -
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: helm-user-cluster-admin-role
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: cluster-admin
+subjects:
+- kind: ServiceAccount
+  name: default
+  namespace: kube-system
+EOF
+```
+
+Create namespace and service account for WebLogic Kubernetes Operator
+
+```bash
+$ kubectl create namespace sample-weblogic-operator-ns
+namespace/sample-weblogic-operator-ns created
+
+
+$ kubectl create serviceaccount -n sample-weblogic-operator-ns sample-weblogic-operator-sa
+serviceaccount/sample-weblogic-operator-sa created
+```
+
+Install Operator
+
+```bash
+$ helm install weblogic-operator kubernetes/charts/weblogic-operator \
+  --namespace sample-weblogic-operator-ns \
+  --set serviceAccount=sample-weblogic-operator-sa \
+  --set "enableClusterRoleBinding=true" --wait
+
+NAME: weblogic-operator
+LAST DEPLOYED: Tue Nov 17 09:33:58 2020
+NAMESPACE: sample-weblogic-operator-ns
+STATUS: deployed
+REVISION: 1
+TEST SUITE: None
+```
+
+Verify the operator with the following command; the status will be running.
+
+```bash
+$ helm list -A
+NAME                        NAMESPACE                     REVISION   UPDATED                                 STATUS       CHART                   APP VERSION
+sample-weblogic-operator    sample-weblogic-operator-ns   1          2020-11-17 09:33:58.584239273 -0700 PDT deployed     weblogic-operator-3.0.3
+
+
+$ kubectl get pods -n sample-weblogic-operator-ns
+NAME                                 READY   STATUS    RESTARTS   AGE
+weblogic-operator-775b668c8f-nwwnn   1/1     Running   0          32s
+```
+
+
+#### Create docker image
+
+ - Image creation
```

Comment:
This bulleted item is out of place because there is just one. I suggest that you remove it. (It's not needed.)